### ISSUE TITLE
Controle ne se Redeclenche pas quand on change le contenu

### ISSUE THREAD

**Issue by JulienCarmona** [InseeFr/stromae-dsfr#160]
## Describe the bug

Lorsqu'on a un contrôle de cohérence sur une question, ce dernier ne se déclenche pas la deuxieme foi et passe à la 'reponse suivante meme si on a changé la réponse et que ca devrait toujours déclencher le controle

- [x] I tested it on the storybook and storybook-DSFR and it works !
- [x] I tested on visu V2 and it works !
## To Reproduce

exemple de questionnaire : [m3podcmf.json](https://github.com/user-attachments/files/17829496/m3podcmf.json) en PROD

1. Faire un questionnaire avec une réponse simple de type nombre entre 0 et 100
2. Ajouter un controle de cohérence, ex `nb < 25` -> controle affiché
3. Visionner le questionnaire en Stromae-DSFR et aller à la question
4. taper 10
5. Le controle s'affiche bien
6. corriger l'input et taper 15
7. BUG : on passe à la question suivante

## Expected behavior
On devrait avoir le contrôle qui bloque encore et se redéclenche

## Version where the bug appeared
Stromae : 1.2.3 | Lunatic : 3.4.4

## Browser where the bug was tested

- [ ] Chrome
- [x] Firefox
- [ ] Safari
- [ ] Edge
- [ ] Other (please specify): ******\_\_\_******

## Additional context
Cela semble etre du à une surcouche de stromae-dsfr car les sotrybook de lunatic fonctionnent bien.

**Comment by QRuhier** [comment]
Il y a quand-même une différence fonctionnelle entre Stromae-dsfr et les autres (Lunatic, Lunatic-DSFR, stromae-v2) :
- dans Stromae-dsfr : on n'affiche pas de pop-up, donc pour autoriser le passage à la suite le 2e clic porte le fait de confirmer la saisie malgré le contrôle
- dans tous les autres : qu'on modifie ou non la valeur, on affiche une pop-up proposant de continuer ou corriger

**Comment by JulienCarmona** [comment]
Mais est-ce qu'on est d'accord sur le fait qu'on devrait avoir le contrôle qui se réactive la deuxieme fois ? 

Ou alors c'est le comportement "normal" de DSFR c'est juste qu'on n'avait encore jamais rencontré le pb et dans ce cas, ce n'est pas réellement un bug, mais plus une évolution ?

**Comment by QRuhier** [comment]
Je ne sais pas vraiment ce qui nous a mené au fonctionnement actuel (qui est là depuis le début de Stromae-dsfr), les demandes/contraintes... En tout cas oui actuellement c'est un peu étrange

**Comment by AnneHuSKa** [comment]
@JulienCarmona @laurentC35 @QRuhier : à ce stade le correctif me semble suffisant. On va surement devoir lancer un peu de toilettage des controles en 2025 suite aux controles bloquants / accessibles ou pas etc... donc ne surinvestissons pas à ce stade non ?

**Comment by JulienCarmona** [comment]
Yes ça me, on va en rester là pour ce ticket et on ne crée rien pour l'instant
